plugins/topology/src/components/Topology/TopologySideBar/TopologyResourcesTabPanel.css

```diff
@@ -0,0 +1,3 @@
+.topology-job-pod-ring svg {
```

Comment:
I know we are not consistent with styles across plugins but we need to decide on how to handle it like scoping to plugin level i.e `bs-topology-*` . WDYT

cc @debsmita1